go/Makefile

```diff
@@ -1,5 +1,5 @@
 COINSTACKS := $(wildcard coinstacks/*)
-MAKEFLAGS += -j$(words $(COINSTACKS)) --no-print-directory
```

Comment:
Can we not keep these jobs in parallel by increasing the resources for docker desktop?

Reply:
We can, I just figured that it might be better to run the jobs sequentially since external contributors may run into the same problem that I did and it isn't obvious what the source of the error is. It's your call on this one, but I was thinking that since the time difference between the sequential and parallel builds isn't substantial, it might be better to play it safe.

Reply:
Yeah fair enough. The `$(words $COINSTACKS))` can be removed along with the flag.